### PR TITLE
User comment connect

### PR DESCRIPTION
**Pull request by janko101**
# Changes proposed in this pull request
- Adds UserName of a comment creator before content.
- Hides Update and Delete buttons of a comment when the current user is not the one who created a comment.

### CODE REVIEW

File: frontend/src/components/comments/CommentCard.jsx [hunk 1]
```diff
@@ -23,18 +32,31 @@ export default function CommentCard({ postId, comment, onDeleteClick }) {
     }
   }
 
+  function checkCommentUserEmail() {
```
Comment (eeels22) [inline frontend/src/components/comments/CommentCard.jsx:35]: Could this be refactored to:

`return commentCreatorEmail === currentUser.email;`

File: frontend/src/components/comments/CommentCard.jsx [hunk 2]
```diff
@@ -5,10 +5,14 @@ import CommentsApi from "../../api/CommentsApi";
 // Components
 import CommentUpdateForm from "./CommentUpdateForm";
 
-export default function CommentCard({ postId, comment, onDeleteClick }) {
+export default function CommentCard({ postId, comment, onDeleteClick, user }) {
```
Comment (deciocandido) [inline frontend/src/components/comments/CommentCard.jsx]: Is there a reason to have the user as argument?

**Review by eeels22 — APPROVED** [review]
Looks great Janko. I put a suggestion for refactoring the checkCommentUserEmail function in CommentCard, but I see that everything is working. Great job.

**Review by deciocandido — APPROVED** [review]
Looks great

File: frontend/src/components/comments/CommentCard.jsx (hunk 1 above)
Comment (janko101) [reply frontend/src/components/comments/CommentCard.jsx:35]: Yap...good one..I refactored it :)

File: frontend/src/components/comments/CommentCard.jsx (hunk 2 above)
Comment (janko101) [reply frontend/src/components/comments/CommentCard.jsx]: yes, so we can check if the logged in user is the user who left the comment

**Pull request merged**